src/main/java/com/hades/exceptions/LoginFailureException.java

```diff
@@ -0,0 +1,9 @@
+package com.hades.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.UNAUTHORIZED, reason = "Invalid credentials")
```

Comment:
Ah sim... se não for muita frescura, troca por ``403 FORBIDDEN``, pois foram passadas credenciais, mas as mesmas nao são válidas.